README.md

```diff
@@ -48,17 +48,17 @@ wizAssets.deleteFile("file://documents/settings/img/cards/card001.jpg", successC
 
 ### deleteFiles()
 
-**wizAssets.deleteFiles(Array manyURIs, Function success, Function fail );**
+**wizAssets.deleteFiles(Array localURIs, Function success, Function fail );**
 
-- delete all URIs in Array like; [ "file://documents/settings/img/cards/card001.jpg", "file://documents/settings/img/cards/card002.jpg " .. ] <br />
-- if you do not specify a filename only dir, then all contents of dir will be deleted; file://documents/settings/img/cards <br />
+- delete all URIs in Array like; [ "img/cards/card001.jpg", "img/cards/card002.jpg " .. ] <br />
+- if you do not specify a filename only dir, then all contents of dir will be deleted; img/cards <br />
```

Comment:
The grammar here is quite funky. Perhaps we can rephrase this? localURI is not even supposed to be "filename". It's a URI, right? :)